modules/mod_ginger_collection/templates/page.collection_object.tpl

```diff
@@ -24,6 +32,11 @@
                     {% include "collection/depiction.tpl" record=record template="collection/download.tpl" %}
                 </div>
             </div>
+
+            {% block prevnext %}
+                {% include "collection/prevnext.tpl" search="beeldenzoeker" %}
```

Comment:
Shouldn’t this be `search="ginger_collection"`?

Reply:
i dont think so since search-query.tpl also uses "beeldenzoeker". This block should be overriden for every website that uses their own version. (as is done for zuiderzeecollectie)